Added support for Hetzner Cloud (hcloud) provider.

seifertm opened the pull request:
I am using this to generate Ansible inventory files for servers provisioned in [Hetzner Cloud](https://hetzner.cloud) using the [HCloud plugin](https://www.terraform.io/docs/providers/hcloud/r/server.html) for Terraform.

I am not familiar with development in GoLang, so please review this with all your scrutiny :)

adammck commented:
Sorry about the delay getting to this. It looks fine to me-- could you please rebase against master?

seifertm commented:
@adammck I rebased it and changed everything to `twenty` or `10.0.0.20` to account for the other integrations that have been added in the meantime. Can you have another look?

DanielMSchmidt commented:
Is there anything we can do to move this forward?

marco-m commented:
In my understanding, this can be closed due to #151. @b-m-f is this correct ?

DanielMSchmidt commented:
@DanielMSchmidt  I can give it a try today

b-m-f commented:
I believe it can be closed, yeah
